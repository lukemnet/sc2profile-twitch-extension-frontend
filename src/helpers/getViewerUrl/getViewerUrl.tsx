import getApiUrl from 'src/helpers/getApiUrl/getApiUrl';

const getViewerUrl = (channelId: string) =>
  getApiUrl({ channelId, type: 'viewer', activity: 'get' });

export default getViewerUrl;