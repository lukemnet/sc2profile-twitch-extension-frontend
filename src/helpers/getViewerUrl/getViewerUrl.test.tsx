import getViewerUrl from 'helpers/getViewerUrl/getViewerUrl';

jest.mock('src/helpers/getApiUrl/getApiUrl', () =>
  jest.fn().mockImplementationOnce(() => ({
    url: 'testViewerUrl',
    method: 'TEST_VIEWER_METHOD',
  })));

it('returns correct viewer API URL', () => {
  expect(getViewerUrl('123')).toMatchSnapshot();
});
