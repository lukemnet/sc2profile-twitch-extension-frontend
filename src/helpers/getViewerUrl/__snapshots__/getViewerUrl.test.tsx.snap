// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns correct viewer API URL 1`] = `
Object {
  "method": "TEST_VIEWER_METHOD",
  "url": "testViewerUrl",
}
`;
